samples/helpers/get-env/src/get_env.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "get_env.hpp"
+
+#if defined(_MSC_VER)
+#if !defined(WIN32_LEAN_AND_MEAN)
+#define WIN32_LEAN_AND_MEAN
+#endif
+#if !defined(NOMINMAX)
+#define NOMINMAX
+#endif
+#include <vector>
+
+#include <windows.h>
+#else
+#include <stdlib.h>
+#endif
+
+std::string GetEnv(const char* name)
+{
+  if (name != nullptr && name[0] != 0)
```

Comment:
switch to indent-less by returning on this condition like

```cpp
if (name == nullptr || name[0] == 0 ) {
  return std::string();
}

// 
```

Reply:
then there is another return will be needed, for the error case. Empty string gets returned if name is empty, or if there's something wrong inside the condition.

Reply:
I rewritten the code, this no longer applies anyways.